components/Footer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-footer class="footer">
+    <div class="copyright">
+      Copyright &copy; {{ new Date().getFullYear() }} COPS
+    </div>
+    <div class="logo">
+      <img src="~/assets/images/new-cops-logo.jpg" width="60px" height="60px" />
+    </div>
+    <div class="link">
+      <a
+        target="_blank"
+        href="https://github.com/COPS-IITBHU/Code-Of-Conduct/blob/master/README.md"
+        >Code of Conduct</a
+      >
+    </div>
+  </v-footer>
+</template>
+
+<script>
+export default {}
+</script>
+
+<style scoped>
+.footer {
+  justify-content: center;
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+}
+.link a {
+  margin: 10px;
+  text-decoration-line: none;
+  color: #fff;
```

Comment:
margin can be added in class ma-1, for link a we can use inline style attribute for
```html
style="text-decoration: none; color: #fff"
```